test/k8s-integration/main.go

```diff
@@ -205,8 +205,8 @@ func runTests(k8sDir string) error {
 	if err != nil {
 		return err
 	}
-	testArgs := "--test_args=--ginkgo.focus=CSI\\sdriver:.*gcePD-external"
-	cmd := exec.Command("go", "run", "hack/e2e.go", "--", "--check-version-skew=false", "--test", testArgs)
+	testArgs := "--test_args=--ginkgo.focus=CSI.*gcePD-external --ginkgo.skip=BlockVolume|\\[Serial\\]|kubelet.*down"
```

Comment:
We don't need to skip BlockVolume anymore now that it's beta.

Reply:
should be fixed in master branch then cherry-picked back to v0.3.0. I'm not going to modify these commits for now

Reply:
oh looks like it has been changed